.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 os : osx
```

Comment:
Actually don't need osx I did it in order to install thrift .. another thing is to just use linux and somehow install thrift